kubeflow/kubeflow
# kf_lang implementation

This PR implements an environment variable named KFLang (Kubeflow Language). 
KF_LANG is an environment variable to be used by notebook servers and is selected by the user upon notebook creation. KF_LANG follows ISO 639-1 formatting (`en`, `fr`). To make full use of KF_LANG each image must have the respective language pack installed and the image scripts modifying the locale environment variable to match KF_LANG. The notebook server will then be configured with the appropriate language.
Related to #5858

To test the feature:
- Build the environment as usual
- Select a language from the dropdown 
- Create a notebook server
- Verify KF_LANG is written to the notebook spec under spec/template/spec/containers/env
